### PR TITLE
🎉 Source Twitter: add missing fields and add authors stream

### PR DESCRIPTION
**Pull request by lideke**
## What
*Retrieve all available fields from the search tweets API

## How
*In the previous version only mandatory fields (id, text...) were configured in the manifest in this version several fields have been added as entities, attachments, reply_settings.

### CODE REVIEW

**Comment by lideke** [comment]
Hi @marcosmarxm, 
I've made some clean up on our branch so why i've closed the existing one.

Thanks in advance for your help

File: airbyte-integrations/connectors/source-twitter/source_twitter/schemas/authors.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
+  "$schema" : "http://json-schema.org/draft-07/schema#",
+  "type" : "object",
+  "properties" : {
+    "id" : {
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-twitter/source_twitter/schemas/authors.json:5]: Make sure the schema is following the ref: https://docs.airbyte.com/connector-development/schema-reference

File: airbyte-integrations/connectors/source-twitter/source_twitter/schemas/authors.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    "username" : {
+      "type" : [
+        "null",
+        "object"
+      ]
+    },
+    "verified_type" : {
+      "type" : [
+        "null",
+        "object"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-twitter/source_twitter/schemas/authors.json:57]: For type objects you can add empty `properties: {}`

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Some comments.

File: airbyte-integrations/connectors/source-twitter/Dockerfile [hunk 3]
```diff
@@ -34,5 +34,5 @@ COPY source_twitter ./source_twitter
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.2
+LABEL io.airbyte.version=0.1.3
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-twitter/Dockerfile:37]: You're adding a new stream so it's a minor update: `0.2.0`

**Pull request closed without merging**